src/oviators/OviatorsExchangeMinterModule.sol

```diff
@@ -138,17 +166,14 @@ contract OviatorsExchangeMinterModule is
     }
 
     function tokenURI(uint256 tokenId) external view returns (string memory) {
-        string memory color = idToColor[tokenId];
-        ColorInfo storage colorInfo = colors[color];
-
         return
             NFTMetadataRenderer.createMetadataEdition({
-                name: string(abi.encodePacked(sink.name(), " ", color)),
+                name: "Oviators", // Token ID is added by NFTMetadataRenderer
                 description: description,
-                imageURI: colorInfo.imageURI,
-                animationURI: colorInfo.animationURI,
+                imageURI: string(abi.encodePacked(imagesBase, tokenId, ".png")),
+                animationURI: string(abi.encodePacked(rendererBase, tokenId)),
```

Comment:
sanity checking: is there an extension needed at the end for the animation URI?

Reply:
no, since the pattern is as follows: `ipfs://bafybeidwdlcqc2eidjqq7afzy3hctmpav3fnrf7g2zhatvznfaoktixvoe/?id=9`.

so the rendererBase would be `ipfs://bafy...voe/?id=`